### PR TITLE
Add putio extension

### PR DESCRIPTION
**Pull request by altaywtf**
## Description

👋 The first ever version of [put.io](https://put.io)'s Raycast extension is here!

Here's what you can do with it: https://github.com/putdotio/putio-raycast/blob/main/CHANGELOG.md#initial-version---2023-06-04

If you need credentials for a demo account, please reach out to `altay@put.io`.

## Screencast

https://github.com/raycast/extensions/assets/9790196/afb5e0ee-05e2-4ee1-875b-d415b63224b4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/putio/package.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "putio",
+  "title": "put.io",
```
Comment (altaywtf) [inline extensions/putio/package.json:4]: this is our branding

**Comment by pernielsentikaer** [comment]
Since you're working for put.io @altaywtf, it might be better to go with this instead of #5979 - what do you think @EpaL 🙂

**Comment by EpaL** [comment]
Sure, @PernielsenTikaer. The work by @AltayWTF looks great! I appreciate that it includes several features I hadn't incorporated into mine. However, I believe it would greatly benefit from having configurable commands to process files. This is particularly crucial for me, given my custom workflow for downloading files from put.io. I imagine other users might find this useful as well. If you're open to it, I could contribute this through a pull request.

**Comment by pernielsentikaer** [comment]
That sounds like a really good idea @altaywtf 🔥 

Is this ready for the hopefully last review?

**Comment by altaywtf** [comment]
yes sir @pernielsentikaer! 😊

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Take a look at what I added, you can maybe do this for the other commands too.

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by altaywtf** [comment]
@pernielsentikaer I guess we're ready for anothoer review.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**